packages/api/src/controllers/asset.ts

```diff
@@ -61,7 +61,10 @@ function defaultObjectStoreId(
   if (isOldPipeline) {
     return config.vodObjectStoreId;
   }
-  if (body.playbackPolicy?.type === "lit_signing_condition") {
+  if (
+    body.playbackPolicy?.type === "lit_signing_condition" ||
+    body.playbackPolicy?.type === "webhook"
+  ) {
```

Comment:
Hmmmmmmm perhaps this should be 
```suggestion
const policyType = body.playbackPolicy?.type;
if (policyType && policyType !== "public") {
```

WDYT??